libcontainer/configs/mountpropagation_linux.go

```diff
@@ -0,0 +1,11 @@
+// +build linux
```

Comment:
this isn't build tag, it should be separated from `package` by empty line.
Better just remove it, you use postfix filename notation anyway.